Gruntfile.js

```diff
@@ -34,6 +66,62 @@ module.exports = function (grunt) {
             dist: {
                 src: ['components/fc.js', 'components/utilities/*.js', '<%= meta.componentsJsFiles %>'],
                 dest: 'dist/<%= pkg.name %>.js'
+            },
+            visualTests: {
+                options: {
+                    sourceMap: true
+                },
+                src: 'visual-tests/src/site/assets/js/**/*.js',
+                dest: 'visual-tests/dist/assets/index.js',
+            }
+        },
+
+        copy: {
```

Comment:
It would be possible to link directly to the assets in `/node_modules` and `/dist`, rather than copying them, but generating the links is trickier.

Copying the files doesn't take long, the files are fairly small (< 2MB) and it makes linking to these files significant easier. I also like that the generated directory, `/visual-tests/dist/`, has no dependencies above itself it the directory structure, so it can be served as the web root.

Happy to change this though, if you think it's worth it?